packages/serverpod/lib/src/server/serverpod.dart

```diff
@@ -370,13 +370,28 @@ class Serverpod {
 
         if (commandLineArgs.applyRepairMigration) {
           logVerbose('Applying database repair migration');
-          await migrationManager.applyRepairMigration(session);
+          var success = await migrationManager.applyRepairMigration(session);
+          if (success) {
+            stdout.writeln('Database repair migration applied.');
+          } else {
+            stderr.writeln('Failed to apply database repair migration.');
+          }
           await migrationManager.initialize(session);
         }
 
         if (commandLineArgs.applyMigrations) {
           logVerbose('Applying database migrations.');
-          await migrationManager.migrateToLatest(session);
+          var migrationsApplied =
+              await migrationManager.migrateToLatest(session);
+
+          if (migrationsApplied == null) {
+            stdout.writeln('Latest database migration already applied.');
+          } else if (migrationsApplied == 1) {
+            stdout.writeln('Database migration applied.');
+          } else {
+            stdout.writeln('$migrationsApplied database migrations applied.');
+          }
```

Comment:
Suggestion: It might be helpful to print the version(s) we applied.

(None blocking comment).

Reply:
Good idea! Will pipe that information out to the user instead.

Reply:
Updated the PR and updated the images 👍